support-frontend/assets/helpers/user/userActions.js

```diff
@@ -24,7 +24,7 @@ export type UserSetStateActions = {|
   setLastName: string => Action,
   setFullName: string => Action,
   setEmail: string => Action,
-  setStateField: string => Action,
+  setStateField: string => (Action | (Function => void)),
```

Comment:
for clarity this line should now be moved to the end, below the comment about page state